_data/projects: in-toto=>adopted

SantiagoTorres opened the pull request:
in-toto has been adopted in production by a couple of companies. Update
the description of some of our latest adoptions, and update the status
of the project from sprouting to adopted.

lukpueh commented on _data/projects.yml:
```diff
@@ -158,16 +158,11 @@ demo video</a> is available."
-    or to clone our <a href=\"https://github.com/in-toto/in-toto/\">repository</a> and follow the
-    directions to integrate in-toto into your software project!"
+      products: "The in-toto software has already been integrated into several open
+  source projects. Most recently, Datadog announced the use of <a href=\"https://www.datadoghq.com/blog/engineering/secure-publication-of-datadog-agent-integrations-with-tuf-and-in-toto/\">TUF and in-toto</a> into their agents integration downloader, and a <a href=\"https://salsa.debian.org/reproducible-builds/debian-rebuilder-setup\">constellation of rebuilders</a> are generating in-toto metadat ...
```
A minor language nit-pick from a non-native speaker:  is *"into"* the right preposition when Datadog *"announced the use of ... into ..."*?

lukpueh replied:
I'd also separate the datadog and rebuilder phrases. I at first thought that dd announced the rebuilder constellation.

SantiagoTorres commented:
> Thanks for updating. Please consider addressing comments below. Also, it might be worth adding a link to https://in-toto.github.io/integrations.html. IMO there is more than datadog and debian to talk about here.

There is, but there's limited space, so I want to keep things succint
